examples/go-client/client.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"log"
+	"path/filepath"
+
+	"github.com/gravitational/teleport"
+	"github.com/gravitational/teleport/lib/auth"
+	"github.com/gravitational/teleport/lib/utils"
+)
+
+func connectClient() (*auth.Client, error) {
+	// set up client TLS authentiction between TLS client and Teleport Auth server.
+	tlsConfig, err := setupClientTLS(context.Background())
+	if err != nil {
+		log.Fatalf("Failed to setup TLS config: %v", err)
```

Comment:
do not use log.Fatal if you can return error, log.Fatal does not give caller a chance to recover - it exists the process